surefire-api/src/test/java/org/apache/maven/surefire/api/util/RunOrderCalculatorTest.java

```diff
@@ -59,4 +62,57 @@ public void testOrderTestClasses()
     {
 
     }
+
+    static class DubboLazyConnectTest
+    {
+
+    }
+
+    public void testOrderTestMethods()
+    {
+        RunOrderParameters runOrderParameters = new RunOrderParameters( "testorder" , null );
+        System.setProperty( "test", "MyGoodTest#a2d,MyGoodTest#aBc,MyGoodTest#abc,MyGoodTest#a1b" );
+        DefaultRunOrderCalculator runOrderCalculator = new DefaultRunOrderCalculator( runOrderParameters, 1 );
+        Comparator<String> testOrderRunOrderComparator = runOrderCalculator.comparatorForTestMethods();
+        String[] strArray = { "abc(MyGoodTest)", "a1b(MyGoodTest)", "a2d(MyGoodTest)", "aBc(MyGoodTest)" };
+        List<String> list = Arrays.asList( strArray );
+        list.sort( testOrderRunOrderComparator );
+        String[] strArray2 = { "a2d(MyGoodTest)", "aBc(MyGoodTest)", "abc(MyGoodTest)", "a1b(MyGoodTest)" };
+        List<String> list2 = Arrays.asList( strArray2 );
```

Comment:
Minor problem but please rename the expected list as expected and actual list as actual